README.md

```diff
@@ -80,18 +77,21 @@ The HTTP library can be used on the browser via the [BrowserClient][] class in
 XMLHttpRequests. For example:
 
 ```dart
+import 'dart:async';
 import 'package:http/browser_client.dart';
-import 'package:http/http.dart' as http;
 
-var client = new BrowserClient();
-var url = "/whatsit/create";
-client.post(url, body: {"name": "doodle", "color": "blue"})
-    .then((response) {
-  print("Response status: ${response.statusCode}");
-  print("Response body: ${response.body}");
-});
+Future<Null> main() async {
```

Comment:
The convention that we have been following on the angular.io side has been to explicitly declare return types. In this case I find it quite useful for the return type to call out to a speed reader that `main` is `async`.

/cc @kwalrath @thso